pages/life-insurance/totally-disabled-or-terminally-ill.md

```diff
@@ -11,17 +14,24 @@ relatedlinks:
     links:
     - url: /life-insurance/manage-your-policy/
       title: Access your policy online
-      description: Need to file a benefits claim after your family member has died? Access your policy online.
-
+      description: Need to file a benefits claim? Access your policy online.
+aliases:
+  - /life-insurance/disabled-and-terminally-ill/
```

Comment:
`aliases` is actually not supported anymore. Do we need a server-side redirect for this?

Reply:
@ncksllvn thanks. Yes, we should do a server side redirect.

 I can remove this alias line from the file.

Reply:
Thanks! If you can, please write a ticket for the server-side redirect and assign me!

Reply:
@ncksllvn ok, will do!